clients/lsp-php.el

```diff
@@ -385,13 +385,8 @@ already present."
 
 (lsp-register-client
  (make-lsp-client
-  :new-connection (lsp-stdio-connection
-                   (lambda ()
-                     (unless lsp-php-composer-dir
-                       (setq lsp-php-composer-dir (lsp-php-get-composer-dir)))
-                     (unless lsp-phpactor-path
-                       (setq lsp-phpactor-path (f-join lsp-php-composer-dir "vendor/phpactor/phpactor/bin/phpactor")))
```

Comment:
It seems to me that you should change:


> (unless lsp-phpactor-path
                       (setq lsp-phpactor-path (f-join lsp-php-composer-dir "vendor/phpactor/phpactor/bin/phpactor")))

to 

> (unless lsp-phpactor-path
  (setq lsp-phpactor-path 
        (or (executable-find "phpactor")
            (f-join lsp-php-composer-dir "vendor/phpactor/phpactor/bin/phpactor"))))


That way you will allow customizing lsp-phpactor-path when phpactor is not on the path. WDYT?

Reply:
I agree with the sentiment, if that's the desired behaviour surely

```lisp
(or lsp-phpactor-path
    (executable-find "phpactor"))
```

makes most sense?

Reply:
You should try lsp-php-composer-dir as well if executable-find does not return, right?

Reply:
... but yes

Reply:
Yes I can make that the fallback but if the first two checks fail that likely will too, but equally there's no reason _not_ to add. I'll commit the requested change now :)